pkg/sloop/store/typed/eventcounttable_test.go

```diff
@@ -101,7 +73,7 @@ func Test_EventCount_PutThenGet_SameData(t *testing.T) {
 }
 
 func Test_EventCount_TestMinAndMaxKeys(t *testing.T) {
-	db, rt := helper_update_eventcount_table(t)
+	db, rt := helper_update_ResourceEventCountsTable(t, (&EventCountKey{}).SetTestKeys(), (&EventCountKey{}).SetTestValue())
```

Comment:
because the test is not looking at the stored value, please use `ResourceEventCounts{}` instead of `(&EventCountKey{}).SetTestValue()`

Reply:
this appears to be the case for numerous tests

Reply:
depending upon the number of uses left for `SetTestValue()`, perhaps it can be removed in favor of a locally constructed `ResourceEventCount{}`

Reply:
I make `SetTestValue` to include only `ResourceEventCount{}`